src/styles/variables/border.js

```diff
@@ -0,0 +1,10 @@
+export const borderRadius = {
+  fullRadius: '100%',
+  partialRadius: '50%',
+  noRadius: '0%',
+  edgeRadius: '4rem',
+};
+
+export const boxShadow = {
+  slightshadow: '0px 4px 1px rgba(0, 0, 0, 0.1)',
```

Comment:
This varies from element to element I think it should be on the element that implements it